zip-1014.rst

```diff
@@ -0,0 +1,438 @@
+parties, and satisfy similar levels of public scrutiny.
+
+All substantial software whose development was funded by the Dev Fund should
+be released under an Open Source license (as defined by the Open Source
+Initiative), preferably the MIT license.
+
+
+Enforcement
+~~~~~~~~~~~
+
+For grant recipients, these conditions should be included in their contract
+with ZF, such that substantial violation, not promptly remedied, will cause
+forfeiture of their grant funds and their return to ZF.
+
+ECC and ZF MUST contractually commit to each other to fulfill these
+conditions, and the prescribed use of funds, such that substantial violation,
+not promptly remedied, will permit the other party to issue a modified version
+of Zcash node software that removes the violating party's Dev Fund slice, and
+use the Zcash trademark for this modified version. The slice's funds will be
+reassigned to ZF-MG (whose integrity is legally protected by the Restricted
+Fund treatment).
+
+
+Future Community Governance
+---------------------------
+
+Decentralized community governance is used in this proposal via the Community
+Panel in the following places:
+
+1. As input into the Major Grant Review Committee which governs
+   the `ZF-MG slice (Zcash Foundation, for major grants)`_.
+
+2. For changing the `Funding Target and Volatility Reserve`_.
+
+It is highly desirable to develop robust means of decentralized community
+voting and governance --- either by expanding the Community Panel
+or a successor mechanism --- and to integrate them into both of these
+processes, by the end of 2021. ECC and ZF SHOULD place high priority on such
+development and its deployment, in their activities and grant selection.
+
+
+ZF Board Composition
+--------------------
+
+Members of ZF's Board of Directors MUST NOT hold equity in ECC or have current
+business or employment relationships with ECC, except as provided for by the
+grace period described below.
+
+Grace period: members of the board who hold ECC equity (but do not have other
+current relationships to ECC) may dispose of their equity, or quit the Board,
+by 1 November 2021. (The grace period is to allow for orderly replacement, and
+also to allow time for ECC corporate reorganization related to Dev Fund
+receipt, which may affect how disposition of equity would be executed.)
+
+The Foundation SHOULD endeavor to use the Community Panel (or successor
+mechanism) as advisory input for future board elections.
+
+
+Disclosures
+===========
+
+The author is
+
+* a coauthor of the Zerocash_ academic paper underlying Zcash;
+* a technical adviser to the Zcash Foundation;
+* a founding scientist, a shareholder, and formerly a technical adviser to the
+  Electric Coin Company;
+* an academic researcher and adviser to various other organizations.
+
+This proposal is his private opinion and does not represent any of the above.
+
+.. _Zerocash: https://eprint.iacr.org/2014/349
```

Comment:
The Disclosures section needs to be removed since it was specific to Eran Tromer being the author. I will do that.

Reply:
just did it in my latest commit @daira (and changed the acknowledgement section slightly)